packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -687,7 +687,7 @@ class _DecoyChildState extends State<_DecoyChild> with TickerProviderStateMixin
   Widget _buildAnimation(BuildContext context, Widget? child) {
     return Positioned.fromRect(
       rect: _rect.value!,
-      child: Container(
+      child: DecoratedBox(
```

Comment:
fly-by comment: DecoratedBox and Container are not a 1:1 replacement because a Container with decoration in certain instances inserts extra padding. So, we cannot just do a mechanical replacement here.

See also:
https://github.com/flutter/flutter/blob/cc9ac7d13cccd24d72faab566f9fb59663c3c3d1/analysis_options.yaml#L237